mlflow/sklearn/__init__.py

```diff
@@ -742,23 +750,24 @@ def _log_pretraining_metadata(estimator, *args, **kwargs):  # pylint: disable=un
         # process; accordingly, we avoid logging initial, untuned parameters
         # for these seed estimators.
         should_log_params_deeply = not _is_parameter_search_estimator(estimator)
-        # Chunk model parameters to avoid hitting the log_batch API limit
-        for chunk in _chunk_dict(
-            estimator.get_params(deep=should_log_params_deeply),
-            chunk_size=MAX_PARAMS_TAGS_PER_BATCH,
-        ):
-            truncated = _truncate_dict(chunk, MAX_ENTITY_KEY_LENGTH, MAX_PARAM_VAL_LENGTH)
-            try_mlflow_log(mlflow.log_params, truncated)
```

Comment:
As part of integrating the new `MlflowAutologgingQueueingClient` into MLflow's sklearn autologging integration, I've opted to remove `try_mlflow_log` statements from MLflow Tracking API calls. `try_mlflow_log` was the predecessor to the autologging safety framework offered by `safe_patch`; `try_mlflow_log` was used to wrap any statements that were prone to failure (e.g. MLflow logging) to prevent exceptions from terminating model training. Now that `safe_patch` provides this safety, `try_mlflow_log` is used to wrap operations considered non-essential to the completeness / correctness of an MLflow run. However, parameter logging is clearly essential; an autologging run should not be marked as "finished successfully" (`FINISHED`) unless its parameters have been successfully logged. This also holds for metrics, artifacts, and tags. Longer term, we should significantly reduce / entirely remove instances of `try_mlflow_log` across all of our autologging integrations; this will aid users in identifying failures.

I've removed `try_mlflow_log` from sklearn as part of this PR because it has no equivalent in `MlflowAutologgingQueueingClient`, nor does it need an equivalent there.